flink-runtime/src/main/java/org/apache/flink/runtime/util/EnvironmentInformation.java

```diff
@@ -70,10 +70,9 @@ public static RevisionInformation getRevisionInformation() {
 				revision = propRevision != null ? propRevision : UNKNOWN;
 				commitDate = propCommitDate != null ? propCommitDate : UNKNOWN;
 			}
-		}
-		catch (Throwable t) {
+		} catch (IOException ex) {
```

Comment:
I would like to keep catching `Throwable`, or at least `Exception` here, as this is a method that should not error out in any case.